Author: Ganesan Rajagopal <rganesan@debian.org>
Description: Adds the Python interpreter shebang line to ssh_session.py.

--- a/examples/ssh_session.py
+++ b/examples/ssh_session.py
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 #
 # Eric S. Raymond
 #
